.circleci/verbatim-sources/workflows-ecr-gc.yml

```diff
@@ -14,7 +14,7 @@
       - ecr_gc_job:
             name: ecr_gc_job_for_caffe2
             project: caffe2
-            tags_to_keep: "348,345,336,325,324,315,306,301,287,283,276,273,266,253,248,238,230,213"
+            tags_to_keep: "369,348,345,336,325,324,315,306,301,287,283,276,273,266,253,248,238,230,213"
```

Comment:
I'm not an expert in CI, but I think there are some other places where you need to specify to use this tag (this one probably just prevent the image to be purged). And you need to run a generate script. (iirc https://github.com/pytorch/pytorch/blob/master/.circleci/cimodel/data/caffe2_build_definitions.py#L15)

Reply:
Yeah, @ezyang is helping with all of that! :-)